src/biogeochem/CNFUNMod.F90

```diff
@@ -1608,6 +1619,44 @@ real(r8) function fun_cost_fix(fixer,a_fix,b_fix,c_fix,big_cost,crootfr,s_fix, t
   end if    ! ends up with the fixer or non-fixer decision
   
   end function fun_cost_fix
+
+
+!=========================================================================================
+  real(r8) function fun_cost_fix_Bytnerowicz_noAcc(fixer,nfix_tmin,nfix_topt,nfix_tmax,big_cost,crootfr,s_fix, tc_soisno)
```

Comment:
I love that this is nicely isolated in it's own clean subroutine. This is also something that could have unit or functional testing around it.

Reply:
True. I will assume that testing is outside the scope of the current PR.